build/modules/sdk/build.xml

```diff
@@ -90,7 +90,7 @@
                         src="${dir.binaries}/${name.dist-directory.sdk}/redist/lib/log4j-1.2.14.jar" />
             <zipfileset includes="**"
                         excludes="commons-logging.properties META-INF/MSFTSIG.RSA META-INF/MSFTSIG.SF"
-                        src="${dir.binaries}/${name.dist-directory.sdk}/redist/lib/applicationinsights-core-0.9.5.jar" />
+                        src="${dir.binaries}/${name.dist-directory.sdk}/redist/lib/applicationinsights-core-1.0.4-SNAPSHOT.jar" />
```

Comment:
As far as the SNAPSHOT in the name, we should leave it as is, because this is not an official release and is indeed a 'snapshot'.  

I'm not sure about the rules of putting unsigned jars in the repository, but, since this jar is bundled inside another jar that we do sign, we should be good to go at the eclipse level.

I grabbed those dlls from a released version of app insights, so presumably they are signed already.